chevah/brink
# [#4032] Update brink for py3 compat. Fix pip install in paver.sh.

Scope
=====

I forgot to release brink in previous PR.

This also updates the code in brink to pass the py3 check in pylint


Changes
=======

Published the package.

The pip install was fixed in paver.sh and the build dir matches the one used in brink.

I have updated the code to have py3 test passing.


How to try and test the changes
===============================

reviewers: @brunogola 

the py3 changes are in the last commit

check that changes make sense
